bin/test_runner.rb

```diff
@@ -286,10 +292,9 @@ def check_todo_list
   def run
     ####  main running loop
     while true
```

Comment:
I was not concerned on measuring the minor difference in how ruby coded the use of `while` and `loop`.  I was aiming at the use of the correct tool for the job.  If it is the consensus to not follow best coding practices in our testing code, than I am strongly against it, but that is no reason not to merge.